src/card.css

```diff
@@ -0,0 +1,90 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
```

Comment:
i think it's better to use in the html file
and font-style: normal;  goes by default, so it's useless